draft-irtf-cfrg-hpke.md

```diff
@@ -466,7 +469,8 @@ A "context" encodes the AEAD algorithm and key in use, and manages
 the nonces used so that the same nonce is not used with multiple
 plaintexts. It also has an interface for exporting secret values,
 as described in {{hpke-export}}. See {{hpke-dem}} for a description
-of this structure and its interfaces.
+of this structure and its interfaces. As is standard with AEAD interfaces,
```

Comment:
I would drop this first clause, "As is standard with AEAD interfaces"

Reply:
👍